ssas/blacklist.go

```diff
@@ -32,9 +33,7 @@ func CreateBlacklistEntry(key string, entryDate time.Time, cacheExpiration time.
 		CacheExpiration: cacheExpiration.UnixNano(),
 	}
 
-	db := GetGORMDbConnection()
-	defer Close(db)
-	err = db.Save(&be).Error
+	err = Connection.Save(&be).Error
```

Comment:
It's not super clear where `Connection` comes from since it lives in the top-level `ssas` package. I'll create a ticket to move the packages around so we can say `db.Connection`.